scripts/stagepages.sh

```diff
@@ -5,16 +5,16 @@ set -v
 
 if [ "$TRAVIS_BRANCH" == "master" ]; then
     cp docs/html5/index.html docs/
-    cp openapi/workflow_execution_service.swagger.yaml ./swagger.yaml
+    cp openapi/data_repository_service_schemas.swagger.yaml ./swagger.yaml
     cp -R web_deploy/* .
 elif [ "$TRAVIS_BRANCH" != "gh-pages" ]; then
-  branch=$(echo "$TRAVIS_BRANCH" | awk '{print tolower($0)}')
-  branchpath="preview/$branch"
-  mkdir -p "$branchpath/docs"
-  cp docs/html5/index.html "$branchpath/docs/"
-  cp docs/pdf/index.pdf "$branchpath/docs/"
-  cp openapi/data_repository_service.swagger.yaml "$branchpath/swagger.yaml"
-  cp -R web_deploy/* "$branchpath/"
+    branch=$(echo "$TRAVIS_BRANCH" | awk '{print tolower($0)}')
+    branchpath="preview/$branch"
+    mkdir -p "$branchpath/docs"
+    cp docs/html5/index.html "$branchpath/docs/"
```

Comment:
At some point, we could consider removing the `docs/html5/` folder from any non-`gh-pages` branches. The `index.html` artifact isn't especially useful as a downloadable / version controlled file.

Not critical for now.